frontend/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "contexture",
+  "scripts": {
+    "css-compile": "node-sass --include-path node_modules --precision 6 scss -o css/",
+    "css-compile2": "node-sass --include-path node_modules --output-style compressed --source-map true --source-map-contents true --precision 6 scss -o css/",
```

Comment:
css-compile2 will later be renamed back, right now it makes debugging easier (and faster), same goes for `npm-run-all` not including `css-prefix`

Reply:
What about using some naming style like `css:prod`?